client/app/components/dashboards/DashboardGrid.jsx

```diff
@@ -9,6 +9,7 @@ import cfg from "@/config/dashboard-grid-options";
 import AutoHeightController from "./AutoHeightController";
 import { WidgetTypeEnum } from "@/services/widget";
 
+import "react-resizable/css/styles.css";
```

Comment:
Why import it specifically in here?

Reply:
Right, it's not needed :+1: Forgot to remove after some experiments